src/test/regress/expected/schema_based_sharding.out

```diff
@@ -1439,6 +1439,11 @@ SELECT pg_reload_conf();
 \c - - - :master_port
 SET search_path TO regular_schema;
 CREATE TABLE type_sing(a INT);
+-- errors out because shard_replication_factor = 2
```

Comment:
let's also try to create a tenant table while the replication factor = 2 and observe that it fails, we can use one of those tenant schemas created in this file

```sql
set shard_rep_factor = 2;
create table tenant_10.repl_table(..);
```